manifests/config/file_keystore.pp

```diff
@@ -72,15 +72,15 @@
   $meta_fqpath = "${file_keystorage_dir}/meta/keys/${path}"
 
   exec { "create ${path}_${name} key path":
-    provider => shell,
-    command  => "mkdir -m 775 -p ${key_fqpath}; chown -R ${user}:${group} ${key_fqpath}",
-    creates  => $key_fqpath,
+    path    => ['/usr/bin', '/usr/sbin', '/bin'],
+    command => "mkdir -m 775 -p ${key_fqpath}; chown -R ${user}:${group} ${key_fqpath}",
```

Comment:
if `chown`'s `-R` here is a red-herring, then please use `install -m 755 -o ${user} -g ${group} -d ${key_fqpath}`